modules/behavioural_qc/php/endpoints/behavioural.class.inc

```diff
@@ -0,0 +1,270 @@
+<?php declare(strict_types=1);
+/**
+ * POST request methods.
+ *
+ * Used for populating the behavioural_qc.
```

Comment:
Does behavioural_qc refer to a table? Unless this refers to a specific variable or table etc, the word should be spelled out normally.